microprofile.ls/org.eclipse.lsp4mp.ls/src/test/java/org/eclipse/lsp4mp/services/ApplicationPropertiesDefinitionTest.java

```diff
@@ -26,6 +26,8 @@
  */
 public class ApplicationPropertiesDefinitionTest {
 
+	private static final String PROPERTY_DOCUMENT_NAME = "/microprofile.properties";

```

Comment:
This might not work on Windows

Reply:
Why? I'm in Windows OS and tests are working.